docs/docs/segment-wakatime.md

```diff
@@ -0,0 +1,56 @@
+---
+id: wakatime
+title: Wakatime
+sidebar_label: Wakatime
+---
+
+## What
+
+Shows the tracked time on [wakatime][wt] of the current day
+
+:::caution
+
+You **must** request an API key at the [wakatime][wt] website.
+The free tier for is sufficient. You'll find the API key in your profile settings page.
+
+:::
+
+## Sample Configuration
+
+```json
+{
+    "type": "wakatime",
+    "style": "powerline",
+    "powerline_symbol": "\uE0B0",
+    "foreground": "#ffffff",
+    "background": "#007acc",
+    "properties": {
+        "prefix": " \uf7d9  ",
+        "url": "https://wakatime.com/api/v1/users/current/summaries?start=today&end=today&api_key=API_KEY",
+        "cache_timeout": 10,
+        "http_timeout": 500
+    }
+},
+```
+
+## Properties
+
+- url: `string` - Your Wakatime summaries URL, inclding the API key. Example above. You'll know this works if you can
+- curl it yourself and get a single value. - defaults to ``
```

Comment:
You could add a link to the actual API doc: 
https://wakatime.com/developers#summaries